pkgs/by-name/po/powerstation/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  udev,
+  pciutils,
+  cmake,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "powerstation";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ShadowBlip";
+    repo = "PowerStation";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
Is the new recommendation